content/guides/concepts/cid.md

```diff
@@ -5,18 +5,39 @@ menu:
         parent: concepts
 ---
 
-A *content identifier* is a value that addresses a single piece of content in IPFS. It is mainly a cryptographic hash of the content, but is encoded as a [multihash](https://github.com/multiformats/multihash) and [multicodec](https://github.com/multiformats/multicodec). (Note: older CIDs have a different design — see [version 0](#version-0) below.)
+## Summary
 
-<!-- TODO: explain more of the details of how CID v1 is composed here. -->
+A *content identifier*, or CID, is a label used for addressing content in IPFS. CID's are used as a standard way of pointing to pieces of information.  CID's identify specific pieces of content stored in IPFS.
```

Comment:
Hmmm, I feel kind of like each of these sentences is just telling me the same thing again. This would be fine if it was just the first sentence and you dropped the rest. (Side note: “label” is a great term here! Wish I’d thought of that 😄)

> CID's

Please don’t use an apostrophe here, it’s not technically correct grammar.

Reply:
Maybe this more redundant than necessary, but the point is to ensure understanding comes across.  I'll take another stab.